### DISCUSSION TITLE
Allow notifications to never time out

### DISCUSSION THREAD

**Discussion by dospunk** [emmanuelsw/notiwind#12] [General]
It would be useful if there were a way to make notifications only close when the user clicks the close button

**Comment by emmanuelsw** [comment]
Hi, I love the idea, could you share more details of how you would like the implementation at API level?

**Reply by dospunk** [reply]
I can think of a couple ways to do it

1. Add a `noTimeout` property to the notification options
    ```js
    notify({
      group: 'foo',
      title: "Success",
      text: "Your account was registered!",
      noTimeout: true,
    })
    ```

2. Allow the user to pass something like `-1`, `undefined`, or `null` to the timeout argument
    ```js
    notify({
      group: "foo",
      title: "Success",
      text: "Your account was registered!"
    }, -1)
    ```

Personally I think the second way is preferable, while it may be a little less clear it prevents the user from putting in both the noTimeout property and a custom timeout and then wondering why it isn't working, plus I feel like using `-1` to mean forever is pretty common. 

JavaScript also has the `Infinity` value, which may be more descriptive but I think is less well known

**Reply by emmanuelsw** [reply]
Released in 2.0.1.

**Comment by krydos** [comment]
The second option looks great. 
Currently, in our project we have to put things like `99999999` in there to keep the notification always (almost) visible.

I've prepared the PR here - https://github.com/emmanuelsw/notiwind/pull/39 that implements `-1` as infinite timeout. 
Let me know what do you think.
